src/odemis/acq/stitching/test/tiledacq_test.py

```diff
@@ -95,77 +111,54 @@ def test_get_number_of_tiles(self):
 
         # Smaller than FoV => 1x1
         tiled_acq_task = TiledAcquisitionTask(self.fm_streams, self.stage,
-                                              area=(0, 0, fov[0] / 2, fov[1] / 2),
+                                              (0, 0, fov[0] / 2, fov[1] / 2),
                                               overlap=0, future=model.InstantaneousFuture())
 
-        num_tiles, starting_pos = tiled_acq_task._getNumberOfTiles()
-        self.assertEqual(num_tiles, (1, 1))
+        num_tiles, _, _ = tiled_acq_task._getNumberOfTiles()
+        self.assertEqual(num_tiles, 1)
```

Comment:
I think it would make sense for all the tests in this test case to actually test the indices returned.